configCivcraft.yml

```diff
@@ -10952,6 +11017,440 @@ recipes:
         item:
           material: JUKEBOX
           amount: 1
+
+#Reinforcement recipes:
+  Produce_basic_granite_Miner_Component:
+    type: PRODUCTION
+    production_time: 2s
+    name: Produce basic Granite Miner Component
+    input:
+      stone:
+        material: STONE
+        amount: 128
+      granite:
+        material: STONE
+        amount: 64
+        durability: 1
+      coalore:
+        material: COAL_ORE
+        amount: 24
+    output:
+      basicMiner:
+        material: FLINT
+        name: Miner Component
+        lore: Basic Reinforcement Ingredient
+        amount: 8
```

Comment:
Should only produce 1, yeah? Same for other Basics. Otherwise this recipe is not only faster but 8x more efficient.